Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -237,15 +294,16 @@ void FieldProbe::ComputeDiags (int step)
         amrex::IndexType const Bytype = By.ixType();
         amrex::IndexType const Bztype = Bz.ixType();
 
-        //defined for use in determining which CPU contains the particles
-        int probe_proc = -1;
-
         // loop over each particle
         // TODO: add OMP parallel as in PhysicalParticleContainer::Evolve
+
+        long numparticles = 0; // particles on this MPI rank
         using MyParIter = FieldProbeParticleContainer::iterator;
         for (MyParIter pti(m_probe, lev); pti.isValid(); ++pti)
         {
             const auto getPosition = GetParticlePosition(pti);
+            auto const np = pti.numParticles();
+            numparticles += np;
```

Comment:
Increasing `numparticles` by `np` at the end of the `MyParIter` loop will make things easier for `m_data` access

Reply:
no longer problematic. See above